### DISCUSSION TITLE
아이템 사용질문

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#966] [5. Q&A]
제가 만든 월드에서 커스텀 아이템을 판매/구매 하게 만들고 싶은데, (Zepeto Unity Custom 환경입니다.)
1. 월드내에 유저 아이템을 저장할 수 있는 방법이 있을까요? 인벤토리를 만들고 싶은데 실제 아이템이 어떻게 저장되는지 모르겠습니다.
2. 실제 화폐 Zem을 이용해서 구매할 수 있을까요>? IWP 아이템을 이용한 튜토리얼은 봤는데, 커스텀 월드에서 젬을 이용한 구매가 가능한가요?
3. 아이템 구매 이후에 월드를 나갔다오면 데이터가 남아있나요 아니면 서버에 저장하는 작업이 필요한가요?>

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 멀티플레이 월드를 구현하신 후 IWP 상품을 구매하도록 하고, 구매 정보를 Datastorage에 저장해서 사용할 수 있습니다.
아래 가이드 및 영상이 도움이 될 것 같습니다.  
https://youtu.be/d-MaqjP-vjA
https://studio.zepeto.me/guides/multiplay-game-datastorage

월드 내에서 판매되는 IWP상품은 모두 젬을 통해서만 구매됩니다.
감사합니다.
